sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharp/CSharpExplodedGraph.cs

```diff
@@ -524,6 +529,14 @@ protected override void VisitInstruction(ExplodedGraphNode node)
                     // Do nothing
                     break;
 
+                case SyntaxKindEx.DiscardPattern:
+                    break;
+
```

Comment:
If discard pattern is not added as instruction in the CFG, then we could remove this part.